src/detect-flowbits.c

```diff
@@ -70,6 +74,39 @@ void DetectFlowbitsRegister (void)
     DetectSetupParseRegexes(PARSE_REGEX, &parse_regex, &parse_regex_study);
 }
 
+static char **StrSplit(char *arrptr, const char *delim)
+{
+    char **strarr = NULL;
+    char *token;
+    int i = 0;
+    if (strchr(arrptr, ' ') != NULL) {
+        SCLogError(SC_ERR_INVALID_SIGNATURE, "Space not allowed in flowbits.");
+        goto error;
+    }
+    strarr = SCCalloc((MAX_TOKENS + 1), sizeof(char **));
+    if (strarr == NULL)
+        goto error;
+    while((token = strtok_r(arrptr, delim, &arrptr))) {
```

Comment:
I think with `strtok_r` you're not supposed to use the same pointer in the 1st and last argument.

Reply:
style: space between `while` and `((`